webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -367,12 +372,13 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
      *
      * @param administrativePermissionIri the IRI of the administrative permission.
      * @param requestingUser              the requesting user.
+     * @param apiRequestID                the API request ID.
      * @return a single [[AdministrativePermissionADM]] object.
      */
-    private def administrativePermissionForIriGetRequestADM(administrativePermissionIri: IRI, requestingUser: UserADM): Future[AdministrativePermissionForIriGetResponseADM] = {
-
-        // FIXME: Check user's permission for operation (issue #370)
-
+    private def administrativePermissionForIriGetRequestADM(administrativePermissionIri: IRI,
+                                                            requestingUser: UserADM,
+                                                            apiRequestID: UUID
+                                                           ): Future[AdministrativePermissionForIriGetResponseADM] = {
```

Comment:
Still needs a check if the user is project admin in the project that the permission is about.

Reply:
Please see my comment below. If necessary, I will add the check to messages in another [PR]